### ISSUE TITLE
Adding listener to filechooser not working

### ISSUE THREAD

**Issue by iMichka** [mgarin/weblaf#260]
Hi

When I add a listener to a filechooser, it is never triggered. I wanted to listen to FILE_FILTER_CHANGED_PROPERTY, but I realized the problem is the same whatever the property. I added an example below. Without WebLaf, propertyChange() will print the name of the property each time something is changed in the Gui. With WebLaf, propertyChange is never called.

```
import java.awt.Container;
import java.awt.EventQueue;
import javax.swing.GroupLayout;
import javax.swing.JComponent;
import javax.swing.JFrame;
import javax.swing.JFileChooser;

import java.beans.PropertyChangeEvent;
import java.beans.PropertyChangeListener;

import com.alee.laf.WebLookAndFeel;
import com.alee.laf.filechooser.WebFileChooser;

import javax.swing.filechooser.FileNameExtensionFilter;

public class FileChooser extends JFrame {

    public FileChooser() {
        initUI();
    }

    private void initUI() {

        WebLookAndFeel.install();

        JFileChooser saveChooser = new JFileChooser();
        saveChooser.setDialogType(JFileChooser.SAVE_DIALOG);
        System.out.println(saveChooser);

        saveChooser.addChoosableFileFilter(new FileNameExtensionFilter("Microsoft Word Documents", "docx"));
        saveChooser.addChoosableFileFilter(new FileNameExtensionFilter("PDF Documents", "pdf"));
        saveChooser.addChoosableFileFilter(new FileNameExtensionFilter("Microsoft Excel Documents", "xlsx"));

        saveChooser.addPropertyChangeListener(listener);

        createLayout(saveChooser);

        setTitle("Save");
        setSize(800, 800);
        setLocationRelativeTo(null);
        setDefaultCloseOperation(EXIT_ON_CLOSE);
    }

    private final PropertyChangeListener listener = new PropertyChangeListener() {
        @Override
        public void propertyChange(PropertyChangeEvent event) {
            System.out.println(event.getPropertyName());
        }
    };

    private void createLayout(JComponent... arg) {

        Container pane = getContentPane();
        GroupLayout gl = new GroupLayout(pane);
        pane.setLayout(gl);

        gl.setAutoCreateContainerGaps(true);

        gl.setHorizontalGroup(gl.createSequentialGroup()
                .addComponent(arg[0])
        );

        gl.setVerticalGroup(gl.createSequentialGroup()
                .addComponent(arg[0])
        );
    }

    public static void main(String[] args) {

        EventQueue.invokeLater(new Runnable() {
        
            @Override
            public void run() {
                FileChooser ex = new FileChooser();
                ex.setVisible(true);
            }
        });
    }
}
```

**Comment by mgarin** [comment]
It looks like those events are thrown directly by the interface elements when something is changed. It is true that I haven't yet implemented this in the file chooser.

I will add improvements so that all events (and probably even some additional ones) will be thrown properly with the same property names as the default ones.

**Comment by mgarin** [comment]
Just a small note - this issue is fixed in a pre-release build available on the site.
That fix will also be included in the final v1.29 release.
